indexer/src/handler.rs

```diff
@@ -0,0 +1,62 @@
+use anyhow::Result;
+use fuel_indexer::EntityResult;
+use fuel_indexer_schema::type_id;
+use serde::{Deserialize, Serialize};
+
+pub type Handle = fn(Vec<Vec<u8>>) -> Result<Option<EntityResult>>;
```

Comment:
- Might need some help here
- Ideally, this would be 
```rust
pub type Handle = fn(Vec<Vec<u8>>) -> Result<Option<Box<dyn Entity>>>;
``` 

OR 

```rust
pub type Handle = fn(Vec<Vec<u8>>) -> Result<Option<Box<T>>>;
```

- But since `T: Entity` isn't a safe object, I get a compiler error